### PR TITLE
patch(Card): Fix Storybook knobs; allow component children

### PR DESCRIPTION
**Pull request by davemn**
## Summary

Update the story for the `<Card>` component to correctly reference its `shadow` prop. Also adds a story for a card with non-text children.

## Test Plan

`yarn test`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ This file is similar to the format suggested by [Keep a CHANGELOG](https://githu
 
 ## Unreleased
 
+- [Patch] Fix **Card** story, relax prop-type check for children ([#1194](https://github.com/optimizely/oui/pull/1194))
```
Comment (ayeshakmaz) [inline CHANGELOG.md]: 🙌

File: src/components/Card/Card.story.js [hunk 2]
```diff
@@ -15,13 +17,33 @@ stories
...
         title={ text('title', 'Card Title') }
-        shadow={ boolean('onClose', false) }
+        shadow={ boolean('shadow', false) }
+        onClose={ action('close card') }
         testSection="card">
         { text('body', 'Hello! This is a short card example.') }
       </Card>
     );
+  }))
+  .add('Custom Contents', (() => {
```
Comment (ayeshakmaz) [inline src/components/Card/Card.story.js]: Love this addition, thank you!!

File: src/components/Card/index.js [hunk 3]
```diff
@@ -56,7 +58,10 @@ const Card = ({
 
 Card.propTypes = {
   /** Elements that appears within the component */
-  children: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([
```
Comment (ayeshakmaz) [inline src/components/Card/index.js]: Great update

**Comment by davemn** [comment]
> This is a great update, thank you! Could you add a test for this new case? Otherwise, looks great!

Can do 👍

**Pull request merged**